### ISSUE TITLE
the "p" of parameters in the API left sidebar is off 

### ISSUE THREAD

**Issue by 12rambau** [jbms/sphinx-immaterial#289]
with respect to the other item types, the "p" is vertically off.

<img width="80" alt="image" src="https://github.com/jbms/sphinx-immaterial/assets/12596392/6ac17f49-f0e0-470f-ba5a-51a8e74a2f46">

I thinks it's because you use both "height" and "line-height" parameters for the "obj-info" icons. as the small "p" goes under the line, it's off in the 16px square. 

I'll open a PR to simply change small "p" into upper "P" which will be centered as the others !

**Comment by jbms** [comment]
I agree that the p doesn't display optimally but having a mix of cases was intentional.  You can customize the letters with this option:

https://jbms.github.io/sphinx-immaterial/apidoc/index.html#objconf-toc_icon_text

**Comment by 2bndy5** [comment]
could also use custom CSS to adjust the colored bg size:
```css
.objinfo-icon__sub-data {
  padding-bottom: 17px; // or whatever you're happy with
}
```

**Comment by 2bndy5** [comment]
fix available in [v0.11.8](https://pypi.org/project/sphinx-immaterial/0.11.8/)
